languages/wordpress-github-sync-zh_CN.po

```diff
@@ -0,0 +1,218 @@
+msgid "The GitHub host to use. Can be changed to support a GitHub Enterprise installation."
+msgstr "GitHub服务器的主机名，企业账户可按实际作出改动。"
+
+#: lib/admin.php:48
+msgid "Repository"
+msgstr "项目名"
+
+#: lib/admin.php:51
+msgid "The GitHub repository to commit to, with owner (<code>[OWNER]/[REPOSITORY]</code>), e.g., <code>benbalter/benbalter.github.com</code>. The repository should contain an initial commit."
+msgstr "GitHub上的项目名，格或为(<code>[作者]/[项目名]</code>)，例如<code>benbalter/benbalter.github.com</code>。项目库必须至少包含一个实始提交。"
+
+#: lib/admin.php:56
+msgid "Oauth Token"
+msgstr "Oauth授权码"
+
+#: lib/admin.php:59
+msgid "A <a href='https://github.com/settings/tokens/new'>personal oauth token</a> with <code>public_repo</code> scope."
+msgstr "GitHub上的<a href='https://github.com/settings/tokens/new'> 个人Oauth授权码</a>，授权范围为<code>public_repo</code>。"
+
+#: lib/admin.php:64
+msgid "Webhook Secret"
+msgstr "Webhook密匙"
+
+#: lib/admin.php:67
+msgid "The webhook's secret phrase."
+msgstr "Webhook回调密匙。"
+
+#: lib/admin.php:72
+msgid "Default Import User"
+msgstr "默认导入人"
+
+#: lib/admin.php:75
+msgid "The fallback user for import, in case WordPress <--> GitHub Sync cannot find the committer in the database."
+msgstr "如果WordPress <--> GitHub在数据库中无法查找到提交者，将默认以该用户作为文章的作者。"
+
+#: lib/admin.php:108
+msgid "Export to GitHub started."
+msgstr "已开始导出到GitHub。"
+
+#: lib/admin.php:115
+msgid "Export to GitHub failed with error:"
+msgstr "导出到GitHub过程中遇到错误中止。"
+
+#: lib/admin.php:122 lib/export.php:201 lib/export.php:235
+msgid "Export to GitHub completed successfully."
+msgstr "成功导出到GitHub。"
+
+#: lib/admin.php:134 views/options.php:9
+msgid "WordPress <--> GitHub Sync"
+msgstr "WordPress <--> GitHub Sync"
+
+#: lib/admin.php:135
+msgid "GitHub Sync"
+msgstr "GitHub同步"
+
+#: lib/cli.php:51 lib/cli.php:93
+msgid "Invalid user ID"
+msgstr "无效的用户ID"
+
+#: lib/cli.php:57 wp-github-sync.php:182
+msgid "Starting full export to GitHub."
+msgstr "正在准备全部导出到GitHub。"
+
+#: lib/cli.php:62
+msgid "Exporting post ID to GitHub: %d"
+msgstr "正在导出到GitHub的文章ID：%d"
+
+#: lib/cli.php:68
+msgid "Invalid post ID"
+msgstr "无效文章ID"
+
+#: lib/cli.php:98 wp-github-sync.php:194
+msgid "Starting import from GitHub."
+msgstr "正在准备从GitHub导入。"
+
+#: lib/client/base.php:84
+msgid "WordPress-GitHub-Sync needs an auth token. Please update your settings."
+msgstr "WordPress-GitHub-Sync需要指定一个授权码，请更新设置。"
+
+#: lib/client/base.php:93
+msgid "WordPress-GitHub-Sync needs a repository. Please update your settings."
+msgstr "WordPress-GitHub-Sync需要指定一个项目库，请更新设置。"
+
+#: lib/client/base.php:102
+msgid "WordPress-GitHub-Sync needs a properly formed repository. Please update your settings."
+msgstr "项目名称有误，请按格式修改。"
+
+#: lib/client/persist.php:23
+msgid "There were no changes, so no additional commit was added."
+msgstr "因为没有更新，所以未添加新的提交(Commit)。"
+
+# I guess have to leave this string 'untranslated'? As I came across this error message which was shown on GitHub if a sync unsuccessful (in payload unsuccessful response).
```

Comment:
You can translate this string. All of these error messages can be returned to the payload, and if it's in your native language, it would be easier to understand what went wrong.